arch/xtensa/include/xtensa_asm2_s.h

```diff
@@ -160,20 +160,20 @@
  * Does not populate or modify the PS/PC save locations.
  */
 .macro ODD_REG_SAVE
-	rsr.SAR a0
+	rsr.sar a0
```

Comment:
Which header is colliding?  Our own symbols should all be namespaced.  This sounds to me like we're pulling in some HAL code that we really don't want...

Reply:
modules/hal/xtensa/zephyr/soc/[board]/xtensa/config/specreg.h